src/main/java/com/jpr/maintenance/graphql/GraphQLUtils.java

```diff
@@ -4,38 +4,50 @@
 import graphql.GraphQLError;
 import graphql.GraphqlErrorBuilder;
 import graphql.execution.DataFetcherResult;
-import graphql.schema.DataFetchingEnvironment;
 import io.vavr.control.Either;
-import org.springframework.dao.DataAccessException;
 import reactor.core.publisher.Mono;
 
 import java.util.concurrent.CompletableFuture;
 import java.util.function.Function;
 
-import static com.jpr.maintenance.validation.errors.InputValidationError.DATA_PERSISTENCE_ERROR;
-
 public class GraphQLUtils {
```

Comment:
the try/catch was not doing anything for me, all exceptions were caught by the default datafetcherexceptionmapper, so i refactored it to use our own.
hope it doesn't break all your stuff... ;)